lib/workers/repository/dependency-dashboard.ts

```diff
@@ -115,7 +115,7 @@ export async function ensureDependencyDashboard(
   ) {
     if (getGlobalConfig().dryRun) {
       logger.info(
-        'DRY-RUN: Would close Dependency Dashboard ' +
+        'DRY-RUN: Would close Dependency Dashboard: ' +
           config.dependencyDashboardTitle
```

Comment:
```suggestion
        { title: config.dependencyDashboardTitle },
        'DRY-RUN: Would close Dependency Dashboard'
```

We should use logger metadata here.

Reply:
sure, even better 👍